crystalBox.py

```diff
@@ -119,7 +121,39 @@ def loadCif(self):
         self.gamma_orig = self.gamma
 
         DeleteWorkspace(Workspace='tmp')
+    
+    def loadCifNormUiso(self):
```

Comment:
I think it will be useful to retain the the original Uisos and corresponding Fsq/Int. More generally, I think it would be good to allow users to specify any arbitrary Uisos and update Fsq/Int. Now I think about it, it would be useful to allow users free control of x,y,z,f too. 

I would do this by retaining the original LoadCif, then create a new function `overrideCrystalStructure` (?) that, optionally, either calls your estimate_uiso or accepts user-specified values, then applies and recalculates the Fsq/Int

The original crystal structure should be kept and we probably need a `resetCrystalStructure` function to put things back as they were. Similar to the existing `resetLattice`